changelog/front-commerce-2.18.mdx

```diff
@@ -0,0 +1,111 @@
+
+As of 2.18.0, this module is now in the Front-Commerce core. New features and improvements will follow [our usual release cadence](https://developers.front-commerce.com/docs/appendices/release-process) and predictability for integrators and merchants.
+
+![Before 2.17.X (included): final version 1.0.0 for the the Front-Commerce Prismic module](./assets/Untitled.png)
+
+Before 2.17.X (included): final version 1.0.0 for the the Front-Commerce Prismic module
+
+![From 2.18.0 (included): the module is now in the Front-Commerce core](./assets/prismic.png)
+
+From 2.18.0 (included): the module is now in the Front-Commerce core
+
+## About Front-Commerce Cloud
+
+In August we migrated every front-end hosted by Front-Commerce to our new infrastructure, Front-Commerce Cloud. They are all now up and running 😎
+
+This allows merchants to benefit from auto-scaling (environments automatically add resources to handle the load), better reliability, and better monitoring and support!
+
+And in regards to our support offer, we are working on a nice interface so that you will be able to monitor your infrastructures, deal with environment variables and quickly access our support team: more to come (very) soon 👀
+
+## Improvements in our payment modules
+
+With the release of Magento 2.4.4, we had to adapt parts of our modules to ensure compatibility: that was the case for [our Adyen connector](https://developers.front-commerce.com/docs/advanced/payments/adyen)
+
+For our [PayPal](https://developers.front-commerce.com/docs/advanced/payments/paypal) and our [Lyra](https://developers.front-commerce.com/docs/advanced/payments/payzen) integrations, we worked on payments IPN improvements (Instant Payment Notification) ****to provide asynchronous validation for embedded payments and other improvements from customer feedback. Want to know more about it? Read [the documentation about payments with Front-Commerce!](https://developers.front-commerce.com/docs/category/payments)
+
+# Other changes
+
+### Bug Fixes
+
+- **CSS:** avoid loading CSS twice
+- **Downloadable-products:** page crash if the customer has no product
+- **image:** wrong condition on presets
+- **Magento2:** Added description in shipping method objects
+- **Magento2:** propagate additional headers in Admin HTTP calls
+- **PWA:** load manifest.json from base url not asset base url
+- **RMA:** properly created date with the corresponding timezone
+- **Routes:** don't loose dynamic routes with same name than subdirectory
+- **Routes:** prioritize more specific children routes of a dynamic path
+- **Social login:** merge guest and user cart on login with Magento2
+
+### Features
+
+- **BigCommerce:** add an image proxy to serve resized and optimized images
+- **BigCommerce:** reset my password
+- **GraphQL:** remove FRONT_COMMERCE_WEB_UNSAFE_USE_DEPRECATED_FIELDS env. Gsupport
```

Comment:
what did you mean here?

Reply:
me nothing :) I'm only the copy pasting :robot: 

joke aside, there's an extra `G` :)

Reply:
Ah yes, that's simple in fact…

Reply:
```suggestion
- **GraphQL:** remove FRONT_COMMERCE_WEB_UNSAFE_USE_DEPRECATED_FIELDS env. support
```